README.md

```diff
@@ -36,17 +38,53 @@ IPFS daemons are already easy to start and stop, but this module is here to do i
 // IPFS_PATH will point to /tmp/ipfs_***** and will be
 // cleaned up when the process exits.
 
-var ipfsd = require('ipfsd-ctl')
+const ipfsd = require('ipfsd-ctl')
 
-ipfsd.disposableApi(function (err, ipfs) {
-  ipfs.id(function (err, id) {
+ipfsd.disposableApi((err, ipfs) => {
+  ipfs.id((err, id) => {
     console.log(id)
     process.kill()
   })
 })
 ```
 
-If you need want to use an existing ipfs installation you can set `$IPFS_EXEC=/path/to/ipfs` to ensure it uses that.
+The daemon controller safely spawns the node for you and exposes you the ipfs API. __If the parent process exits, the daemon will also be killed__ ensuring that the daemon isn't left hanging.
+
+This module works by downloading the binary once, on first use, if it detects that no current binary is available to use. So keep in mind that the first command executed might throw in some overhead.
+
+If you want to use an existing ipfs installation you can set `$IPFS_EXEC=/path/to/ipfs` to ensure it uses that.
+
+## API
+
+## ipfsd
+
+#### ipfsd.version(done)
+
+#### ipfsd.local(path, done)
+
+#### ipfsd.disposableApi(opts, done)
+
+#### ipfsd.disposable(opts, done)
+
+## Ipfs(path, opts, disposable)
+
+#### ipfs.init(initOpts, done)
+
+#### ipfs.shutdown(done)
+
+#### ipfs.startDaemon(done)
+
+#### ipfs.stopDaemon(done)
+
+#### ipfs.daemonPid()
+
+#### ipfs.getConfig(key, done)
+
+#### ipfs.setConfig(key, value, done)
+
+#### ipfs.replaceConf(file, done)
+
+#### ipfs.version(done)
 
```

Comment:
To be honest, I would like to use:
`ipfsd.start()`
`ipfs.stop()`

and that's it. start() would automatically init a repo if one doesn't exist at the given path.

However, I reckon that this would require a bigger re-think of the library and as I don't want to bikeshed it too much, I'm happy to use the current API if that's an easier way to bring this to a working state.